Objective-C wrapper for multimodal LLM runner.

shoumikhin opened the pull request:
Summary: .

Differential Revision: D81936485

jackzhxng reviewed (approved):
I think we are going to get rid of `RawAudio` and just have the runner always take processed audio

shoumikhin committed:
Objective-C wrapper for multimodal LLM runner. (#14075)

Summary:
Pull Request resolved: https://github.com/pytorch/executorch/pull/14075

.

Reviewed By: jackzhxng

Differential Revision: D81936485
